packages/components/tooltip/__tests__/tooltip.test.tsx

```diff
@@ -16,7 +16,7 @@ const AXIOM = 'Rem is the best girl'
 
 describe('<ElTooltip />', () => {
   const createComponent = (props: any = {}, content: string | VNode = '') =>
-    mount(Tooltip, {
+    mount(<Tooltip />, {
```

Comment:
**Suggestion**

This is a non-blocking comment. 😊

```ts
  mount(
    <Tooltip
      {...props}
      v-slots={{
        default: () => AXIOM,
        content: () => content,
      }}
    />,
    {
      attachTo: document.body,
    }
  )
```

And line#18: `props: any = {}` -> `props = {}`